lib/tasks/enable_block_level_sync.rake

```diff
@@ -0,0 +1,17 @@
+desc "Enable block level sync for users"
+task enable_block_level_sync: :environment do |_t, args|
+  # bundle exec enable_block_level_sync[user_id_1,user_id_2,...]
+
+  user_ids = args.extras
+  Rails.logger.info user_ids.map { |user_id|
```

Comment:
@prabhanshuguptagit can you inline this logger, so we get separate logs for each user?